### PR TITLE
SDS client: ignore <128 bytes MSEED files

### PR DESCRIPTION
**Pull request by megies**
Just ignore small MSEED files, in use cases working with near-realtime data these are usually just being created right at request time, e.g. when fetching current data or checking latency right after midnight when first samples are currently being written to a new file.

### CODE REVIEW

**Comment by megies** [comment]
We already have `obspy.core.util.obspy_types.ObsPyException` and I'm already using it e.g. in the SDS report script and Reftek-130 read plugin.. 😜 

But yeah, good idea to also do it here.. since we're just now adding these new exceptions..

**Comment by krischer** [comment]
Ah I did not even know that :)

If you end up changing it here - make sure the `ObsPyMSEEDReadingError` or how its called also inherits from the generic miniseed exception.

**Comment by megies** [comment]
I've tried to come up with some proposal for more fine-grained exceptions, see also #639 and feel free to comment or modify..

File: obspy/io/mseed/headers.py [hunk 1]
```diff
@@ -53,6 +53,9 @@
 VALID_RECORD_LENGTHS = [256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536,
                         131072, 262144, 524288, 1048576]
 
+FILESIZE_TOO_SMALL_MSG = (
+    'The smallest possible mini-SEED record is made up of 128 bytes.')
+
```
Comment (krischer) [inline obspy/io/mseed/headers.py]: This is no longer needed and can be directly folded in the the error message.

**Pull request merged**